index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio Project</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h1 class="title">Welcome!</h1>
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
+    </nav>
+    <section class="headline">
+        <div class="intro-txt">
+            <h2 class="wave-txt">Hey there. I'm Lucas</h2>
+            <h2 class="desc-txt">I'm a software developer</h2>
+        </div>
+        <div class="main-headline-txt">
+            <p>
+            I am begginer student at Microverse, currently learning how to implement flexbox properties in CSS. This is
+            my first project using flexbox! Below are the social media links to contact me
+            </p>
+        </div>
+        <div class="media-imgs">
+            <img src="imgs/twitter.svg" alt="Twitter img">
+            <img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
+            <img src="imgs/medium.svg" alt="Medium img">
+            <img src="imgs/github.svg" alt="Github img">
+            <img src="imgs/angellist.svg" alt="Angellist img">
+        </div>

```

Comment:
- [ ] Well done, you added all social media Icons but please use the `ul` tag to group your social media icons, and `li` per icon, and put each icon inside a link ` a.